YSE_App/data_ingest/QUB_data.py

```diff
@@ -25,6 +25,7 @@
 from django.db.models import Q
 from YSE_App.models import Transient, TransientTag
 import sys
+from tendo import singleton
```

Comment:
DJ -- did you add a new package to your local environment? I checked yse_virtual, and I don't see this dependency on prod. If this requires a new package, you'll also need to update the requirements.txt. Next week PM and I will be happy to help with that process.